src/static/css/product-card.scss

```
@@ -0,0 +1,197 @@
+.product-information {
+    display: flex;
+    margin-bottom: 100px;
+
+    @media screen and (max-width: #{$breakpoint-tablet - 1}) and (min-width: $breakpoint-mobile){
+        display: block;
+    }
+
+    @media only screen and (max-width: #{$breakpoint-desktop - 1}) and (min-width: $breakpoint-tablet) {
+        //tablet
+        display: block;
+    }
```

you could do this so you dont repeat code 
```suggestion
    @media screen and (max-width: $breakpoint-desktop){
        display: block;
    }
```
and you could use `flex-direction: column;` , so you keep working with flex